src/etc/sysctl.d/30-vyos-router.conf

```diff
@@ -108,3 +108,5 @@ net.ipv4.tcp_congestion_control=bbr
 # Disable IPv6 Segment Routing packets by default
 net.ipv6.conf.all.seg6_enabled = 0
 net.ipv6.conf.default.seg6_enabled = 0
+
+net.vrf.strict_mode = 1
```

Comment:
I don't see this option (VRF is not configured in my case)
```
vyos@r4:~$ sysctl net.vrf.strict_mode 
sysctl: cannot stat /proc/sys/net/vrf/strict_mode: No such file or directory
vyos@r4:~$ 
vyos@r4:~$ show version 
Version:          VyOS 1.5-rolling-202402060022
Release train:    current
```

Reply:
It will only appear once the VRF Kernel lodule is loaded.

Thats the case why the Kernel was changed to always include it